host/xtest/regression_1000.c

```diff
@@ -2316,3 +2316,26 @@ static void xtest_tee_test_1031(ADBG_Case_t *c)
 ADBG_CASE_DEFINE(regression, 1031, xtest_tee_test_1031,
 		 "Test C++ features");
 #endif
+
+static void xtest_tee_test_1032(ADBG_Case_t *c)
+{
+	TEEC_Result res = TEEC_SUCCESS;
+	TEEC_Context ctx = { };
+	TEEC_SharedMemory shm = {
+		.buffer = xtest_tee_test_1032,
```

Comment:
This is read-only but also executable. How about also testing an initialized static const buffer (thus in the `.rodata` section)?

Reply:
OK